packages/components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -167,7 +167,7 @@ export const Checkbox = React.forwardRef<HTMLButtonElement, CheckboxProps>(
           )}
         </StyledCheckbox>
         <Text.label
-          color={error ? "colorTextError" : "colorTextStronger"}
+          color="colorTextStrongest"
```

Comment:
Are we alos changing from Stronger to Strongest?

Reply:
Yeah, I forgot this. I will update the changeset.